fontc_crater/src/ci/html.rs

```diff
@@ -172,6 +153,27 @@ fn make_table_body(runs: &[RunSummary]) -> Markup {
     }
 }
 
+/// for values that change between runs, make the little decoration span that says like,
+/// "+0.333" or "-5"
+fn make_delta_decoration<T: PartialOrd + Copy + Sub<Output = T> + Display + Default>(
+    current: T,
+    prev: Option<T>,
+    more_is_better: bool,
```

Comment:
Suggest enum instead of bool, the callsites are a bit opaque. A trait might be nice? - `ratio.delta_decoration(prev, Better::More)`

Reply:
added the enum, trait is definitely an option but would rather just get this merged than noodle on it much more. :)